### DISCUSSION TITLE
Local time timestamp?

### DISCUSSION THREAD

**Discussion by kaihendry** [KyleRoss/node-lambda-log#73] [Q&A]
I'm not to JS and I'm actually struggling to work out how I configure the logger to output time stamps in local time, e.g. 

```
$ date -Iseconds
2021-10-15T09:13:22+08:00
```

Do I need to use moment-timezone?

**Comment by KyleRoss** [comment]
LambdaLog does not output the timestamps, they are part of CloudWatch Logs and you are not able to change those timestamps. You'd have to add the timestamp with the `dynamicMeta` function in your local time if you wanted your local timestamp to be included within the log.
